src/universal/modules/userDashboard/components/CreditCardModal/CreditCardField.js

```diff
@@ -1,101 +1,101 @@
-import PropTypes from 'prop-types'
```

Comment:
can remove file?

Reply:
I believe so, didn’t see any import statements for this file.